POC/ColorPicker/ColorPicker/ColorPickingFunctionality/SystemEvents/RegisteredMouseEventHook.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace ColorPicker.ColorPickingFunctionality.SystemEvents
```

Comment:
Hooks can have disastrous impact to other applications on the system. Is a hook really needed here? Consider perhaps using a tiny square window/button that travels with the cursor around the screen, is nearly transparent (e.g. #01000000), can accept your click and route accordingly?

Reply:
Good idea, will look into it

Reply:
This idea is probably the better way of doing, but it is a large change and will be made in a future PR. Thanks for the idea!